### PR TITLE
[#183047939] Add maintenance window retrieval info for redis

### PR DESCRIPTION
**Pull request by whi-tw**
## What

Add info for how to retrieve the maintenance windows for redis service instances.

## How to review

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. Check the content is correct

## Who can review

Not me

### CODE REVIEW

File: source/documentation/deploying_services/redis.md.erb [hunk 1]
```diff
@@ -239,7 +239,20 @@ If you do not have a high availability service plan, you will lose data during a
 
 #### Redis maintenance times
 
-Every Redis service has a maintenance window of Sunday 11pm to Monday 1:30am UTC every week. Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you require a different maintenance window.
+Every Redis service instance has a maintenance window pre-configured. The day and time of this maintenance window can be retrieved from the Cloudfoundry CLI by running:
+
+```
+cf service --params SERVICE_NAME
+```
+
```
Comment (cadmiumcat) [inline source/documentation/deploying_services/redis.md.erb:247]:
```suggestion
Here is an example of the output you will see:
```

**Review by cadmiumcat — COMMENTED** [review]
Looks good 👍. I've added a suggestion
Do we need a tech writer to double check this?

**Review by cadmiumcat — APPROVED** [review]
Very minor change from tech writer. Good otherwise! 👍

**Comment by risicle** [comment]
(otherwise I'd like to copy this block to the postgres section)

**Review by risicle — APPROVED** [review]
*I* like this, should probably double check with a tech writer.

File: source/documentation/deploying_services/redis.md.erb [hunk 2]
```diff
@@ -239,7 +239,22 @@ If you do not have a high availability service plan, you will lose data during a
 
 #### Redis maintenance times
 
-Every Redis service has a maintenance window of Sunday 11pm to Monday 1:30am UTC every week. Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you require a different maintenance window.
+Every Redis service instance has a maintenance window pre-configured. The day and time of this maintenance window can be retrieved with the Cloud Foundry CLI (version 8 and above) by running:
```
Comment (cmcnallygds) [inline source/documentation/deploying_services/redis.md.erb:242]:
```suggestion
Every Redis service instance has a maintenance window pre-configured. You can retrieve the day and time of this maintenance window with the Cloud Foundry CLI (version 8 and above) by running:
```

**Review by cmcnallygds — CHANGES_REQUESTED** [review]
Looks good - have suggested a small change to get rid of passive voice as per the style guide.

**Review by cmcnallygds — APPROVED** [review]

**Pull request merged**